lib/jobs/obm-control.js

```diff
@@ -38,17 +38,17 @@ function obmControlJobFactory(
      * @param {String} taskId
      * @constructor
      */
-    var ObmControlJob = function ObmControlJob(options, context, taskId) {
+   var ObmControlJob = function ObmControlJob(options, context, taskId) {
         ObmControlJob.super_.call(this, logger, options, context, taskId);
 
-        assert.string(this.context.target);
-        assert.string(this.options.action);
+        assert.string(this.options.action, 'action');
         assert.ok(_.contains(_.methods(ObmService.prototype), this.options.action),
                 'OBM action is a known action');
         // Defaults for this should have been set by the definition
-        assert.ok(this.options.obmServiceName);
+        assert.ok(this.options.obmServiceName, 'obm service name');
 
-        this.nodeId = this.context.target;
+        this.nodeId = this.context.target || this.options.nodeId;
```

Comment:
could we add a quick unit test validating this logic? As I'm reading it, the Job will use target by default, and override specifically with `options.nodeId` if it's there. Looks like it would be a quick add to `spec/lib/jobs/obm-control-spec.js`